pool/src/auctions/user_liquidation_auction.rs

```diff
@@ -41,40 +38,45 @@ pub fn create_user_liq_auction_data(
         panic_with_error!(e, PoolError::InvalidLiquidation);
     }
 
+    let percent_liquidated_i128_scaled = i128(percent_liquidated) * position_data.scalar / 100; // scale to decimal form with scalar decimals
+
     // ensure liquidation size is fair and the collateral is large enough to allow for the auction to price the liquidation
     let avg_cf = position_data
         .collateral_base
-        .fixed_div_floor(position_data.collateral_raw, oracle_scalar)
+        .fixed_div_floor(position_data.collateral_raw, position_data.scalar)
         .unwrap_optimized();
     // avg_lf is the inverse of the average liability factor
     let avg_lf = position_data
         .liability_base
-        .fixed_div_floor(position_data.liability_raw, oracle_scalar)
+        .fixed_div_floor(position_data.liability_raw, position_data.scalar)
         .unwrap_optimized();
-    let est_incentive = (SCALAR_7 - avg_cf.fixed_div_ceil(avg_lf, SCALAR_7).unwrap_optimized())
-        .fixed_div_ceil(2 * SCALAR_7, SCALAR_7)
-        .unwrap_optimized()
-        + SCALAR_7;
+    let est_incentive = (position_data.scalar
+        - avg_cf
+            .fixed_div_ceil(avg_lf, position_data.scalar)
+            .unwrap_optimized())
+    .fixed_div_ceil(2 * position_data.scalar, position_data.scalar)
+    .unwrap_optimized()
+        + position_data.scalar;
 
     let est_withdrawn_collateral = position_data
         .liability_raw
-        .fixed_mul_floor(percent_liquidated_i128, oracle_scalar)
+        .fixed_mul_floor(percent_liquidated_i128_scaled, position_data.scalar)
         .unwrap_optimized()
-        .fixed_mul_floor(est_incentive, SCALAR_7)
+        .fixed_mul_floor(est_incentive, position_data.scalar)
         .unwrap_optimized();
     let mut est_withdrawn_collateral_pct = est_withdrawn_collateral
-        .fixed_div_ceil(position_data.collateral_raw, oracle_scalar)
+        .fixed_div_ceil(position_data.collateral_raw, position_data.scalar)
         .unwrap_optimized();
-    if est_withdrawn_collateral_pct > SCALAR_7 {
-        est_withdrawn_collateral_pct = SCALAR_7;
+    if est_withdrawn_collateral_pct > position_data.scalar {
+        est_withdrawn_collateral_pct = position_data.scalar;
```

Comment:
I still think we should update the negative tests to have different oracle and reserve scalars to ensure no bugs could exist in this math